packages/polkadart_scale_codec/test/test_types/codec_u64_test.dart

```diff
@@ -22,14 +22,14 @@ void main() {
     test(
         'Should return correct encoded data when value fits 64 bits and is positive',
         () {
-      final largestSupportedValue = BigInt.parse('18446744073709551615');
+      final largestSupportedValue = '18446744073709551615'.toBigInt;
       const expectedResult = '0xffffffffffffffff';
 
       expect(CodecU64().encodeToHex(largestSupportedValue), expectedResult);
     });
 
     test('Should throw InvalidSizeException when value is negative', () {
-      final value = BigInt.from(-1);
+      final value = (-1).toBigInt;
```

Comment:
this seems more natural, although I don't know if it'll work lol
```suggestion
      final value = -BigInt.one;
```